apps/namadillo/src/App/Masp/ShieldedBalanceChart.tsx

```diff
@@ -73,13 +83,6 @@ export const ShieldedBalanceChart = (): JSX.Element => {
 
   useEffect(() => {
     shieledSync();
-
-    const timeoutId = setTimeout(() => {
```

Comment:
The original idea here is to avoid flashing a "100%" in case it syncs quickly (less than 3s)

Reply:
Yeah! I changed it because I had some problems getting right behavior when navigating from settings. RIght now instead of timeout it does this:
```ts
        // If total is more than one batch of 100, show progress
        if (total > 100) {
          setShowSyncProgress(true);
        }
 ```
so we show progress immediately but only if there is more than 100 blocks to fetch and parse :)